api/pom.xml

```diff
@@ -110,6 +110,33 @@
 				<filtering>true</filtering>
 			</testResource>
 		</testResources>
+
+		<plugins>
+			<plugin>
+				<groupId>com.googlecode.maven-java-formatter-plugin</groupId>
```

Comment:
It is to help in the formatting of code using a standard format style acceptable by the whole team. Just to make our code uniform